### PR TITLE
docs(iconography): page content

### PR DESCRIPTION
**Pull request by Vanessarich**
# General summary

Doc & code can be review.

Note/need help on: 
- The theme switcher doesn't work on "Sizes" section.
- Need to remove "Show code" and "Dark background" options for "layout" section.

# Screenshots
FireShot Capture 037 - Design System - 0 0 0 0

# Check list

-   [ ] (if has style) Add `need: review-integration` label
-   [ ] (if has textual documentation) Add `need: review-doc` label
-   [ ] (if has code) Add `need: review-frontend` label
-   [ ] (if has react) Check through the [react dev check list]
-   [ ] (if fix or feature) Add `need: test` label

Check out the [contribution guide] for general guidelines for submissions to the design system.

[contribution guide]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md
[react dev check list]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md#react-developpment-check-list

### CODE REVIEW

File: demo/react/doc/product/foundations/iconography.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Icons are small visual representations of common actions, objects or ideas. They help users to identify information and provide context.**
+
+## Sizes
+Comes in 5 sizes: xs, s, m, l, xl.
+```javascript jsx withThemeSwitcher
+(theme) => (
+    <>
+        <Icon icon={mdiEmail} theme={theme} size={Size.xs}>
+            Header with icon
```
Comment (gcornut) [inline demo/react/doc/product/foundations/iconography.mdx]: The Icon component doesn't have content.
You can remove the text `Header with icon` and the closing tag `</Icon>`:
```jsx
<Icon icon={mdiEmail} theme={theme} size={Size.xs} />
```

Comment (Vanessarich) [reply demo/react/doc/product/foundations/iconography.mdx]: Done ;)

File: demo/angularjs/foundations/iconography/doc.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Iconography
```
Comment (FlorentDinet) [inline demo/angularjs/foundations/iconography/doc.md]: this file is not up to date

File: demo/react/doc/product/foundations/iconography.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import { mdiEmail } from 'LumX/icons';
+```
+
+
+# Iconography
+
+**Icons are small visual representations of common actions, objects or ideas. They help users to identify information and provide context.**
+
+## Sizes
+Icons come in 5 sizes: xs, s, m, l, xl.
```
Comment (FlorentDinet) [inline demo/react/doc/product/foundations/iconography.mdx]: in 6 sizes: _xxs_, _xs_, _s_, _m_, _l_, _xl_

**Pull request closed without merging**